src/modals/manage-farming-pool-modal/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { RouteProp, useRoute } from '@react-navigation/native';
+import { noop } from 'lodash-es';
+import React, { FC, useEffect, useRef } from 'react';
+import { Text, View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { ButtonLargePrimary } from 'src/components/button/button-large/button-large-primary/button-large-primary';
+import { Divider } from 'src/components/divider/divider';
+import { ModalButtonsContainer } from 'src/components/modal-buttons-container/modal-buttons-container';
+import { ModalStatusBar } from 'src/components/modal-status-bar/modal-status-bar';
+import { ScreenContainer } from 'src/components/screen-container/screen-container';
+import { TextSegmentControl } from 'src/components/segmented-control/text-segment-control/text-segment-control';
+import { useBlockLevel } from 'src/hooks/use-block-level.hook';
+// import { FormAssetAmountInput } from 'src/form/form-asset-amount-input/form-asset-amount-input';
```

Comment:
Please look for commented out code, `console.log` statements and remove them. Sorry for forgetting to remove them myself